<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-02-13T11:52:46 IST" name="Default suite" finished-at="2024-02-13T11:52:55 IST" duration-ms="8090">
    <groups>
    </groups>
    <test started-at="2024-02-13T11:52:46 IST" name="Default test" finished-at="2024-02-13T11:52:55 IST" duration-ms="8090">
      <class name="extentReport.TestScript">
        <test-method is-config="true" signature="configureReports()[pri:0, instance:extentReport.TestScript@35e2d654]" started-at="2024-02-13T11:52:46 IST" name="configureReports" finished-at="2024-02-13T11:52:46 IST" duration-ms="162" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configureReports -->
        <test-method is-config="true" signature="browserSetup()[pri:0, instance:extentReport.TestScript@35e2d654]" started-at="2024-02-13T11:52:46 IST" name="browserSetup" finished-at="2024-02-13T11:52:51 IST" duration-ms="4291" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- browserSetup -->
        <test-method is-config="true" signature="login(java.lang.reflect.Method)[pri:0, instance:extentReport.TestScript@35e2d654]" started-at="2024-02-13T11:52:51 IST" name="login" finished-at="2024-02-13T11:52:52 IST" duration-ms="1506" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void extentReport.TestScript.clickOnBooks()]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="clickOnBooks()[pri:0, instance:extentReport.TestScript@35e2d654]" started-at="2024-02-13T11:52:52 IST" name="clickOnBooks" finished-at="2024-02-13T11:52:53 IST" duration-ms="1202" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Demo Web Shop] but found [Demo Web Shop. Books]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Demo Web Shop] but found [Demo Web Shop. Books]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at extentReport.TestScript.clickOnBooks(TestScript.java:15)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clickOnBooks -->
        <test-method is-config="true" signature="logout()[pri:0, instance:extentReport.TestScript@35e2d654]" started-at="2024-02-13T11:52:54 IST" name="logout" finished-at="2024-02-13T11:52:54 IST" duration-ms="773" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:extentReport.TestScript@35e2d654]" started-at="2024-02-13T11:52:54 IST" name="closeBrowser" finished-at="2024-02-13T11:52:55 IST" duration-ms="158" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method is-config="true" signature="reportBackup()[pri:0, instance:extentReport.TestScript@35e2d654]" started-at="2024-02-13T11:52:55 IST" name="reportBackup" finished-at="2024-02-13T11:52:55 IST" duration-ms="267" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reportBackup -->
      </class> <!-- extentReport.TestScript -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
